src/Pages/Explore/JobSlide/JobSlide.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from "react";
+import SlideItem from "./SlideItem";
+import SlideFliter from "../JobSlide/SlideFliter/SlideFliter";
+import { config } from "./config";
+import styled from "styled-components";
+import JobCard from "./../../../Components/JobCard/JobCard";
+
+const JobSlide = () => {
+  const [datas, setData] = useState([]);
+  useEffect(() => {
+    fetch("http://localhost:8000/companies")
+      .then((res) => res.json())
+      .then((res) => setData(res["category_list"]));
+  }, []);
+
+  useEffect(() => {}, [datas]);
+
+  const [appearCard, setAppear] = useState(0);
+
+  const moveValue = appearCard * 100;
+
+  const slideResult = Math.floor((datas.length - 1) / 10);
```

Comment:
```suggestion
  const [appearCard, setAppear] = useState(0);
  const moveValue = appearCard * 100;
  const slideResult = Math.floor((datas.length - 1) / 10);
```
저 위에 있는 useState랑 붙여주세요